src/FileMapper.ts

```diff
@@ -245,7 +245,7 @@ export class FileMapper implements Disposable {
      * @param artifacts array of sarif.Artifact that needs to be mapped
      * @param runId id of the run these files are from
      */
```

Comment:
I need to make a pass through all of the "public" methods and fix the documentation. Especially one's i've changed.